README.md

```diff
@@ -598,7 +598,8 @@ Http is available as an injectable class, with methods to perform http requests.
 
 #### Developer tools
 
-* [Augury](https://augury.rangle.io/) - Browser extension for debugging and profiling Angular applications. 
+* [Augury](https://augury.rangle.io/) - Browser extension for debugging and profiling Angular applications.
+* [ngx-cdmon](https://www.npmjs.com/package/ngx-cdmon) - Library for monitoring Angular change detection performance.
```

Comment:
I think `ngx-cdmon` is a 3rd party component, please move there.

Reply:
I thought that was only for Angular components/directives (not necessarily services/utility functions/etc.). Anyway, will move it, then.